kit/inputs/src/UIMaterialTextView/hooks/useMaterialTextViewChildren.tsx

```diff
@@ -208,12 +216,18 @@ export function useMaterialTextViewChildren(
 }
 
 const styles = StyleSheet.create({
+    iconTapZone: {
+        height: ICON_TAP_ZONE_SIZE,
+        width: ICON_TAP_ZONE_SIZE,
+        alignItems: 'center',
+        justifyContent: 'center',
+    },
     iconSize: {
         width: UILayoutConstant.iconSize,
         height: UILayoutConstant.iconSize,
     },
-    iconsFiller: {
-        marginLeft: UILayoutConstant.smallContentOffset,
+    iconOffset: {
+        marginRight: -UILayoutConstant.normalContentOffset,
```

Comment:
I'm a bit confused with the negative offset... but will approve it)

@SAnatoliiS @savelichalex what do you think?